labelbox/schema/model_run.py

```diff
@@ -574,21 +574,28 @@ def send_to_annotate_from_model(
             batch_name: str, data_rows: Union[DataRowIds, GlobalKeys],
             params: SendToAnnotateFromModelParams) -> Task:
         """
-        Sends data rows from a model run to a project for annotation.
+        Sends data rows from a model run to the specified project in Annotate either as prelabels or labels.
 
         Example Usage:
             >>> task = model_run.send_to_annotate_from_model(
             >>>     destination_project_id=DESTINATION_PROJECT_ID,
             >>>     batch_name="batch",
-            >>>     data_rows=UniqueIds([DATA_ROW_ID]),
+            >>>     data_rows=DataRowIds([DATA_ROW_ID]), # or GlobalKeys([GLOBAL_KEY])
             >>>     task_queue_id=TASK_QUEUE_ID,
-            >>>     params={})
+            >>>     params={
+            >>>         "predictions_ontology_mapping": {
+            >>>             "feature_schema_id": "feature_schema_id"
```

Comment:
nit: would be best to give customers clear understanding of what these feature schema ids represent.

`"model_run_schema_id": "destination_project_schema_id"`